cmd/release-controller-api/http.go

```diff
@@ -312,6 +313,10 @@ func (c *Controller) userInterfaceHandler() http.Handler {
 	mux.HandleFunc("/api/v1/releasestream/{release}/candidate", c.apiReleaseCandidate)
 	mux.HandleFunc("/api/v1/releasestream/{release}/release/{tag}", c.apiReleaseInfo)
 	mux.HandleFunc("/api/v1/releasestream/{release}/config", c.apiReleaseConfig)
+
+	// static files
+	mux.PathPrefix("/").Handler(http.FileServer(http.Dir("./static")))
+
```

Comment:
Take a look at the `embed` package for this content
https://pkg.go.dev/embed

Reply:
Don't forget this one...

Reply:
Done.